pkg/model/awsmodel/bastion.go

```diff
@@ -77,14 +78,26 @@ func (b *BastionModelBuilder) Build(c *fi.ModelBuilderContext) error {
 
 	for _, src := range bastionGroups {
 		// Allow traffic from bastion instances to egress freely
-		t := &awstasks.SecurityGroupRule{
-			Name:          fi.String("bastion-egress" + src.Suffix),
-			Lifecycle:     b.SecurityLifecycle,
-			SecurityGroup: src.Task,
-			Egress:        fi.Bool(true),
-			CIDR:          fi.String("0.0.0.0/0"),
+		{
+			t := &awstasks.SecurityGroupRule{
+				Name:          fi.String("ipv4-bastion-egress" + src.Suffix),
+				Lifecycle:     b.SecurityLifecycle,
+				SecurityGroup: src.Task,
+				Egress:        fi.Bool(true),
+				CIDR:          fi.String("0.0.0.0/0"),
+			}
+			AddDirectionalGroupRule(c, t)
+		}
+		{
+			t := &awstasks.SecurityGroupRule{
+				Name:          fi.String("ipv6-bastion-egress" + src.Suffix),
```

Comment:
Consider making one `SecurityGroupRule` that allows egress for both IPv4 and IPv6 CIDRs.

Reply:
Seems a good idea for a future PR. Would have to check first if what is simplified is worth the added complexity.

Reply:
It would avoid changing the name of the rule, so an upgrade wouldn't delete and recreate.

Reply:
The rule name doesn't change actually, because it uses `AddDirectionalGroupRule`, that overwrites the name for the task based on rule components. This is why we don't see any changes in TF or CF output either.